vuepress/src/Documentation/Pre-reading-material/000200_sea_at_bc_Gov.md

```diff
@@ -0,0 +1,82 @@
+# SEA at BC Gov
+* AWS SEA at BC Gov
+
+![AWS SEA](./images/sea.png)
+
+* GCP SEA at BC Gov
+
+![GCP SEA](./images/bc_gov_sea_cloud.png)
+
+* The above images shows the relationship between services offered in different scopes of the various contexts of the AWS, GCP SEA. BC Gov teams will be working primarily in the green area with some services from the blue. 
+* The green services have been tested by the Cloud Pathfinder team for good fit within the guardrails of the environment.  
+* Some services are not compatible but in general there is a pattern for teams to use the blue ones if needed.
+## SEA Concepts
+* AWS SEA Concept
+![AWS SEA concepts](./images/aws_sea_concepts.png)
+* GCP SEA Concept
+![GCP SEA concepts](./images/gcp_sea_concept.png)
+
+## Public Cloud SEA Services
+* AWS
+
+![compliant services in aws](./images/compliant_services_aws.png)
+::: tip Acronyms
+- **CPF:** Cloud pathfinder Team
+- **ECS** = Elastic Container Service
+- **EC2** = Elastic Compute Cloud/Virtual Machines
+- **CSP** = Cloud Service Provider
+- **Lambdas** = a serverless technology where code is run on CSP maintained servers, turned off when not in use
+- **IAM** = Identity Access Management of users in the CSP
+- **AI/ML** = Artificial Intelligence and Machine Learning Cloud services 
+  :::
+
+  * GCP
+
+![compliant services in GCP](./images/compliant_services_gcp.png)
+::: tip Acronyms
+- **VMs:** Elastic Compute Cloud Virtual Machines
```

Comment:
I think the GCP VMs likely have a different brand name

Reply:
Yep, Sorry it should be Google Compute Engines.